imgui/core.pyx

```diff
@@ -1374,6 +1374,68 @@ def is_window_collapsed():
     return cimgui.IsWindowCollapsed()
 
 
+def set_tooltip(str text):
+    """Set tooltip under mouse-cursor.
+    Usually used with :func:`is_item_hovered()`.
+    For a complex tooltip window see :func:`begin_tooltip()`.
+
+    .. visual-example::
+        :auto_layout:
+        :height: 100
+        :width: 200
+        :click: 10 10
+
+
+        imgui.begin("Example: tooltip")
+        imgui.button("Click me!")
```

Comment:
Could be `"Hover me!"` because we are displaying action on hovering, not clicking.

In future I will consider adding some `:hover:` option analogous to `:click:` for visual examples generation. Maybe we will even completely redesign how interactions are handled (animated examples?).